toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -382,9 +383,21 @@ func customizeOSContents(ic *ImageCustomizerParameters) error {
 		}
 	}
 
+	ukiEnabled := false
+	if ic.config.OS.Uki != nil {
```

Comment:
`OS` can be nil, we should probably make sure it exists first if we are not already

Reply:
never mind - i expanded the code, and it's guaranteed to exist in this function.